app.js

```diff
@@ -803,6 +803,7 @@ app.all('/jquery.*', ensureRestAuthenticated, preassembleBody, setOpenhab, proxy
 app.all('/classicui/*', ensureRestAuthenticated, preassembleBody, setOpenhab, proxyRouteOpenhab);
 app.all('/ui/*', ensureRestAuthenticated, preassembleBody, setOpenhab, proxyRouteOpenhab);
 app.all('/basicui/*', ensureRestAuthenticated, preassembleBody, setOpenhab, proxyRouteOpenhab);
+app.all('/basicui/*', ensureRestAuthenticated, preassembleBody, setOpenhab, proxyRouteOpenhab);
```

Comment:
didn't you maybe want to put "paperui" in there?
If so, you can actually replace the line "/ui/*" - this was the old uri of the Paper UI, which does not exist anymore.